cmd/tsbs_load_timescaledb/main.go

```diff
@@ -167,5 +176,10 @@ func getConnectString() string {
 		connectString = fmt.Sprintf("%s password=%s", connectString, pass)
 	}
 
+	if forceTextFormat {
+		// we assume we're using pq driver
```

Comment:
You might be able to force text format using the pgx driver by setting `PreferSimpleProtocol`to true in the pgx connection config? Probably not worth changing at this point but maybe if we have more time.

Reply:
Yeah, I though about that. However, copy protocol in PGX only supports binary so we would still need PG...